### ISSUE TITLE
Determine framework(s)

### ISSUE THREAD

**Issue by kevinzurek** [OpenDevGroup/OpenDevManagement#2]
This will probably be a web application, but what frameworks will we use?

**Comment by JustSteveKing** [comment]
I prefer PHP as it is my language of choice, and I use Slim only in my apps as it is micro and lightweight

**Comment by kevinzurek** [comment]
My strengths are .NET MVC and Play!

**Comment by JustSteveKing** [comment]
I guess we are going to have different areas? I mean people in the organisation have different skills after all!

**Comment by kevinzurek** [comment]
Yeah, let's see what everyone likes and make a decision. I'm open to learning something new.

**Comment by JamieAmstel** [comment]
I would be interested in using node js?

**Comment by JustSteveKing** [comment]
I am also open to learning Node - it seems like the future

**Comment by syrano** [comment]
i'd be super interested in node as well

**Comment by kevinzurek** [comment]
I would also be interested in Node, but I wouldn't be comfortable governing pull requests for it. Anyone familiar enough and willing to do so?

**Comment by JustSteveKing** [comment]
I think we need to look at frameworks per language instead of an all over one. That way if you are to pick up a new language and need a framework the group has a kind of default one per language. It will aid in learning new languages and frameworks.

**Comment by kevinzurek** [comment]
Is anyone interested in setting up the foundations for this in Node.js and helping to manage pull requests? Otherwise i can do it in .NET MVC (C#)

**Comment by jasonmcauley** [comment]
We were having a discussion about this in the slack chat. Basically my thoughts are we can try Play to expose people to some Scala/Java or Ruby on Rails (as it is a pretty marketable skill to have nowadays). Basically my interests kind of have me slanted towards Play... But my practical side has me thinking Ruby on Rails. 

My selfishness has me turning away from .NET due to the fact that I work with it all day and would like to learn something new... However I am not completely opposed to it if that is what people want.

**Comment by JamieAmstel** [comment]
http://www.reddit.com/r/opendev/comments/2v7nbb/project_opendevmanagement_languageframework_poll/

**Comment by syrano** [comment]
I'm totally in for Rails actually. I just started with the manual tbh but it' something I really want to get involved with too

**Comment by H4kor** [comment]
I only have experiences in django and my developing experiences are limited to solo projects.

**Comment by joshfarrant** [comment]
I'm a big fan of Ember for the front-end, plus there's Ember Data which can interface with the API. I'd also like to try out React too, although that's purely view as far as I understand. Have a little bit of experience with Node but would love to spend some more time with it too.
